docs/autofish.rst

```diff
@@ -22,16 +23,23 @@ Usage
 ``autofish <max> [min] [<options>]``
     Change autofish settings.
 
-    ``max`` (default: 100) controls the maximum amount of fish you  want to keep
-    on hand in your fortress. Fishing will be disabled when the amount of fish
-    goes above this value.
+Positional Parameters
+---------------------
+
+``max``
+    (default: 100) controls the maximum amount of fish you want to keep on hand
+    in your fortress. Fishing will be disabled when the amount of fish goes
+    above this value.
+
+``min``
+    (default: 75) controls the minimum fish you want before restarting fishing.
 
-    ``min`` (default: 50) controls the minimum fish you want before restarting
-    fishing.
+Options
+-------
 
-    Use ``--toggle-raw``(``-r``) (default: on) to toggle letting the script
-    also count your raw fish as part of your quota. Use it a second time to
-    disable this.
+``-r``, ``--toggle-raw``
```

Comment:
This doesn't have to be changed in this PR (and we're really too close to release to add behavior changes now), but I'm realizing that this is probably not a good choice for an option. If you have a startup script with `autofish -r` in it, is that turning it on or off? What if you have two scripts that both assume they're turning the option in a particular direction? This should probably be changed into `--raw=<boolean>`